snap/snapcraft.yaml

```diff
@@ -54,22 +58,37 @@ parts:
       
       craftctl default
 
+  # Include a copy of clknetsim.so built using the correct glibc, used for testing of the snap
+  clknetsim:
+    plugin: nil
+    source-type: git
+    source: https://github.com/mlichvar/clknetsim.git
```

Comment:
This should be versioned, along with linuxptp-testsuite to match the LinuxPTP version.